packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```diff
@@ -61,5 +61,9 @@
   "pci_database_add_spec_nodes_range_single": "{{min}} nœud inclus, jusqu'à {{max}} nœuds",
   "pci_database_add_show_command": "Voir l'équivalent API",
   "pci_database_add_spec_name_popover": "Le nom de votre cluster est généré automatiquement. Vous pourrez le modifier par la suite.",
-  "pci_database_add_spec_logo_alt": "Logo du moteur"
+  "pci_database_add_spec_logo_alt": "Logo du moteur",
+  "pci_database_add_network_mode_helper": "Pour en savoir plus sur la configuration d'un réseau privé, consultez le guide ",
+  "pci_database_add_network_mode_helper_link": "vRack Public Cloud",
```

Comment:
These sentences should not be split, you can't be sure that all languages will have this form.

Reply:
Hi @JeremyDec, I have to split here because one part of the sentence is a link: 
```
<span data-translate="pci_database_add_network_mode_helper">
</span>
<a
    class="oui-link_icon"
    href="{{::$ctrl.privateNetworkGuideUrl}}"
    target="_blank"
    rel="noopener"
>
    <span
        data-translate="pci_database_add_network_mode_helper_link"
    ></span>
    <span
        class="oui-icon oui-icon-external-link"
        aria-hidden="true"
    ></span>
</a>
  ```

Until then, we always split the sentences like we dit here, but I understand it is not perfect. Maybe adding ":" at the end of the first part would help ? Something like that:
```
"pci_database_add_network_mode_helper": "Pour en savoir plus sur la configuration d'un réseau privé, consultez le guide : ",
  "pci_database_add_network_mode_helper_link": "vRack Public Cloud",
```

Let me know what you think is best here :wink:

Reply:
@jperchoc , you can try passing the required html content as strings using translate-values. 

**startTag** `'<a  class=&quot oui-link_icon &quot href=&quot + $ctrl.privateNetworkGuideUrl + &quot target=&quot _blank &quot >` 

**endTag** `<span class=&quot oui-icon oui-icon-external-link &quot aria-hidden=&quot true &quot ></span> </a>`

In translate file: 

`"Pour en savoir plus sur la configuration d'un réseau privé, consultez le guide : {{ startTag }} vRack Public Cloud {{ endTag }}"`

Reply:
Hi @anooparveti ,
I tried passing the html with translate-values, but it does not seems to work (html tags are automatically closed):

html : 
```
<span 
    data-translate="pci_database_add_network_mode_helper"
    data-translate-values="{
        startTag: '<a class=&quot oui-link_icon &quot href=&quot' + $ctrl.privateNetworkGuideUrl + '&quot target=&quot _blank &quot >',
        endTag: '<span class=&quot oui-icon oui-icon-external-link &quot aria-hidden=&quot true &quot ></span> </a>',          
    }">
</span>
```

translation file: 
```
"pci_database_add_network_mode_helper": "Pour en savoir plus sur la configuration d'un réseau privé, consultez le guide : {{ startTag }} vRack Public Cloud {{ endTag }}",
```

Here's the resulting html in the browser :
![image](https://user-images.githubusercontent.com/24541481/179172052-43fb2be1-7f50-485e-9c18-353e2d961235.png)

As you can see, the `<a>` opened in the `startTag` is closed in the generated html, resulting in a broken link on the page.
Am I forgetting something ?
Thanks !

Reply:
I managed to have something working :
translation file: 
```
  "pci_database_add_network_mode_helper": "Pour en savoir plus sur la configuration d'un réseau privé, consultez le guide : {{ link }}",
  "pci_database_add_network_mode_helper_link": "vRack Public Cloud",
```

html file:
```
<span 
    data-translate="pci_database_add_network_mode_helper"
    data-translate-values="{
        link: 
        '<a 
            class=&quot oui-link_icon &quot 
            href=&quot' + $ctrl.privateNetworkGuideUrl + '&quot 
            target=&quot _blank &quot 
        >
            <span>{{:: 'pci_database_add_network_mode_helper_link' | translate}}</span>
            <span 
                class=&quot oui-icon oui-icon-external-link &quot 
                aria-hidden=&quot true &quot 
            ></span>
        </a>',
    }">
</span>
```

This seems to work fine, I have the link correctly displayed, and it should not be a problem for translation, as the link is directly inserted into the translation string, so it can be placed anywhere depending on the language.

However, I had to replace
```
 <span data-translate=&quot pci_database_add_network_mode_helper_link &quot></span>
```
by 
```
 <span>{{:: 'pci_database_add_network_mode_helper_link' | translate}}</span>
```
fot the link to be displayed.

https://github.com/ovh/manager/pull/7460/commits/f70f728157a04bc69b1fb01a43b18968a76bf827